inst/tests/tests.Rraw

```diff
@@ -21555,7 +21555,9 @@ test(2333, as.expression(data.table(a = 1))[["a"]], 1)
 
 # regression test for hexdigits subscript overrun (uint8_t wraps over 255, unsigned overflow is well defined in c)
 f = tempfile()
-writeLines(c('a', rep('0x1.ffffp0', 10000L), '0x1.ff\x9fp0', rep('0x1.ffffp0', 20000L)), f)
+ff = file(f, encoding = "")
```

Comment:
might as well put the whole thing in `local()` to `on.exit()` closing & deleting.

Also a comment pointing to this PR would be good for posterity sake.